src/e2e-test/java/io/cdap/plugin/utils/E2ETestUtils.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.utils;
+
+import io.cdap.e2e.utils.ConstantsUtil;
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * E2ETestUtils contains the helper functions.
+ */
+public class E2ETestUtils {
```

Comment:
Looks like this class is used in multiple e2e test repos. Instead of duplicating the code it would be better to move it into the e2e test framework